src/components/navBar/navBar.spec.js

```diff
@@ -0,0 +1,247 @@
+describe('<md-nav-bar>', function() {
+  var el, compile, scope, timeout, ctrl, tabContainer, $material, $mdConstant;
+
+  /** @ngInject */
+  var injectLocals = function(
+      $compile, $rootScope, $timeout, _$material_, _$mdConstant_) {
+    compile = $compile;
```

Comment:
If you want, you can keep the $ in the variables names, like $scope, or $timeout.